.github/workflows/test-brew-formulas.yaml

```diff
@@ -0,0 +1,47 @@
+name: test-formulas
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test-formulas:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Brew version output
+        run: brew --version
+      - name: INSTALL bats-core
+        run: brew uninstall bats; brew install bats-core
+      - name: Bats-support AUDIT
+        run: brew audit --new-formula --formula Formula/bats-support.rb
+      - name: Bats-support INSTALL
+        run: brew install --build-from-source --verbose bats-support
+      - name: Bats-support TEST
+        run: brew test Formula/bats-support.rb
+        continue-on-error: true
```

Comment:
Doesn't this defeat the purpose?

Reply:
you mean the `continue-on-error: true` ? yep that was added just to let the other steps running even if we got an error on a previous step, when you will approve the final not-failing tests we can remove that line obv

Reply:
I did get rid of most of the continue on error

the audit command fixes formatting if the homebrew formula is written incorrectly so that has been valueable.

Reply:
You could split the audit and tests into separate jobs.